eq-author-api/docker-compose.yml

```diff
@@ -21,7 +21,8 @@ services:
       - AWS_SECRET_ACCESS_KEY=dummy
       - NODE_ENV=development
       - RUNNER_SESSION_URL=http://localhost:5000/session?token=
-      - PUBLISHER_URL=http://host.docker.internal:9000/publish/
+      - PUBLISHER_URL=http://host.docker.internal:4000/convert/
+      - CONVERSION_URL=http://host.docker.internal:9000/publish/
       - SURVEY_REGISTER_URL=http://host.docker.internal:8080/submit
       - ENABLE_IMPORT=true
       - JAEGER_SERVICE_NAME=eq_author_api
```

Comment:
The gcp feature flag will also need to be added to the docker-compose yml file for v3 to run

Reply:
Done